src/logging/config/LogConfiguration.cpp

```diff
@@ -18,12 +18,13 @@ LogConfiguration::LogConfiguration(
   tagLog.setDocumentation("Configures logging sinks based on Boost log.");
 
   auto attrLogEnabled = makeXMLAttribute("enabled", true)
-                            .setDocumentation("Enables logging");
+                            .setDocumentation("Enables the creation of log sinks. Disable sinks if you plan to setup Boost.log in your application.");
   tagLog.addAttribute(attrLogEnabled);
 
   XMLTag tagSink(*this, "sink", XMLTag::OCCUR_ARBITRARY);
   tagSink.setDocumentation("Contains the configuration of a single log sink, which allows fine grained control of what to log where. "
-                           "Available attributes in filter and format strings are `%Severity%`, `%ColorizedSeverity%`, `%File%`, `%Line%`, `%Function%`, `%Module%`, `%Rank%`, and `%Participant%`");
+                           "Available attributes in filter and format strings are `%TimeStamp%`, `%Runtime%`, `%Severity%`, `%ColorizedSeverity%`, `%File%`, `%Line%`, `%Function%`, `%Module%`, `%Rank%`, and `%Participant%`. "
+                           "The boolean attribute `%preCICE%` is `true` for all log entries originating from preCICE.");
```

Comment:
That's not visible from the user side, is it?

Reply:
An application using Boost.log needs to register sinks.
These sinks will also receive preCICE logs, which is by design of Boost.log.
This application could use this boolean as an easy way to filter out everything from preCICE.

It is not relevant for the vast majority of people, but it could be relevant for some.
I also need to add documentation on this to our docs.